slackify-html.js

```diff
@@ -261,6 +261,21 @@ function walk(dom, nesting) {
             var alt = el.attribs.alt;
             out += '<Inline Image' + (alt !== '' ? '('+alt+')' : '') + ': ' + el.attribs.src + '>';
             break;
+          case 'blockquote':
+            content = walk(el.children);
+            var innerOutput = '';
+            var contentArr = content.split('\n');
+            contentArr.forEach((item) => {
+              if (el.name === 'br' || el.name === 'p') {
+                innerOutput += '>' + item;
+              }
+              innerOutput += '>' + item + '\n';
```

Comment:
is this supposed to be inside an "else" block?

Reply:
yep just updated it